runtime/common/src/evm/precompile.rs

```diff
@@ -54,6 +63,39 @@ where
 	}
 }
 
+pub struct DevelopmentPrecompiles<R>(PhantomData<R>);
```

Comment:
Can we document what this is? I will look further around to understand its application but going through it alone is not clear to me

Reply:
Plus, can we name this just `Development`, given it's defined in the `precompile.rs` module? That way we use it as `precompiles::Development` 👍